developer-extension/src/panel/components/json.tsx

```diff
@@ -44,6 +49,7 @@ const JsonContext = createContext<{
   defaultCollapseLevel: number
   getMenuItemsForPath?: GetMenuItemsForPath
   formatValue: FormatValue
+  sdkType?: 'rum' | 'logs'
```

Comment:
Suggestion: it could be nice to keep the json component more or less generic, because it's used in multiple places, not only for displaying configuration. Maybe add a `onRevealFunctionLocation` prop and externalize that logic?